sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRDD.scala

```diff
@@ -61,7 +61,13 @@ object JDBCRDD extends Logging {
       try {
         val rs = statement.executeQuery()
         try {
-          JdbcUtils.getSchema(rs, dialect)
+          val metaStructType = JdbcUtils.getSchema(rs, dialect)
```

Comment:
Hi  @gatorsmile ，
Thanks for the comments. I will push code changes as you suggested later.